components/layout_2020/display_list/mod.rs

```diff
@@ -291,14 +291,18 @@ impl Fragment {
                         .to_webrender();
                     let common = builder.common_properties(clip, &image.style);
 
-                    builder.wr().push_image(
-                        &common,
-                        rect,
-                        image_rendering,
-                        wr::AlphaType::PremultipliedAlpha,
-                        image.image_key,
-                        wr::ColorF::WHITE,
-                    );
+                    if let Some(image_key) = image.image_key {
+                        builder.wr().push_image(
+                            &common,
+                            rect,
+                            image_rendering,
+                            wr::AlphaType::PremultipliedAlpha,
+                            image_key,
+                            wr::ColorF::WHITE,
+                        );
+                    } else {
+                        builder.wr().push_rect(&common, rect, wr::ColorF::WHITE);
+                    }
```

Comment:
The issue with the poster test came from adding a default white background if there was no poster image. Since Servo is not rendering the SVG's yet, the video background differed from the reference.

![image](https://github.com/user-attachments/assets/d9dee88d-1308-4c00-82e2-330be62f4100)

How it looks in Firefox:

![image](https://github.com/user-attachments/assets/12e27d25-fcf8-4904-9f2b-414872eeddff)